firebase-common/src/main/java/com/google/firebase/annotations/PreviewApi.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.annotations;
+
+import com.google.android.gms.common.annotation.KeepForSdk;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that this object (class, method, etc) is experimental and that both its signature and
+ * implementation are subject to change. An API marked with this annotation provides no guarantee
+ * of API stability or backward-compatibility.
+ */
+@KeepForSdk
+@Target({ElementType.TYPE, ElementType.FIELD, ElementType.METHOD, ElementType.CONSTRUCTOR})
+public @interface PreviewApi {}
```

Comment:
We have a `firebase-annotations` sdk, would it make sense to move it there?

Reply:
I can do this, but I would have to add a dependency on `com.google.android.gms:play-services-basement` to use `@KeepForSdk`.

Reply:
Why do you need `@KeepForSdk`? it has no effect in github

Reply:
Okay, removed "KeepForSdk" and moved the class to firebase-annotations. PTAL.